tmh_registry/registry/api/serializers.py

```diff
@@ -42,9 +41,11 @@ class Meta:
     def to_representation(self, instance):
         data = super(ReadPatientSerializer, self).to_representation(instance)
 
-        data["national_id"] = int(data["national_id"])
-        data["phone_1"] = int(data["phone_1"])
-        data["phone_2"] = int(data["phone_2"])
+        data["national_id"] = (
+            int(data["national_id"]) if data["national_id"] else None
+        )
+        data["phone_1"] = int(data["phone_1"]) if data["national_id"] else None
```

Comment:
@dkoutsonikolis Maybe you meant something like `int(data["phone_1"]) if data["*phone_1*"] else None`? Ideally, I would argue against storing national ID & phones as numbers, but that's another discussion.

Reply:
You are right, this is a mistake due to copy-paste - I will fix it. The national ID and phiones are stored in CharFields.

Reply:
Fixed!